packages/kbn-test/src/functional_test_runner/lib/mocha/reporter/scout_ftr_reporter.ts

```diff
@@ -64,7 +69,9 @@ export class ScoutFTRReporter {
   }
 
   private getFileOwners(filePath: string): string[] {
-    const concatenatedOwners = getCodeOwnersForFile(filePath, this.pathsWithOwners);
+    const owners: CodeOwnership = getCodeOwnersForFile(filePath, this.pathsWithOwners);
+
+    const concatenatedOwners = owners?.teams;
```

Comment:
```suggestion
    const concatenatedOwners = getCodeOwnersForFile(filePath, this.pathsWithOwners)?.teams;
```

There's no need for the `CodeOwnership` type import or the `owners` variable.